Adding Tls client information (#1344)
temporalio/documentation: docs/app-dev-context/set-namespace.md

[Namespaces](/concepts/what-is-a-namespace) are a logical unit of isolation within the Temporal Platform.

> To set a custom Namespace with the Client options in your Temporal Client, you must register your custom Namespace with the Temporal Cluster before setting it in the Temporal Client.

Reviewer: ```suggestion
To set a custom Namespace, register your custom Namespace with the Temporal Cluster first, and then set it in the Temporal Client with the Client Options.
```
